demo/tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "baseUrl": ".",
         "paths": {
             "*": [
+                "./node_modules/tns-core-modules/*",
```

Comment:
I think that `skipLibCheck` resolves another problem which happen to fail with the same error (heap out of memory).